content/pubs/pubs.bib

```diff
@@ -31,22 +75,19 @@ @article{Vogelstein2018equivalence
   year  = {2018},
   url = "https://arxiv.org/abs/1806.05514"
 }
-
 @article{Vogelstein2018coding,
-  author = {Bridgeford, Eric W and Sussman, Daniel and Vince Lyzinski and Vogelstein, Joshua T.},
+  author = {Vogelstein, Joshua T. and Bridgeford, Eric W and Mensh, Brett and Priebe, Carey E.},
```

Comment:
I think this stuff is showing up here, even though it's been fixed on deploy already, because this branch is out of date.  Can you please pull in deploy and push?